news/_posts/2020-09-30-how-to-write-a-new-ui-test.md

```diff
@@ -0,0 +1,376 @@
+---
+layout: post
+title:  "Automated UI tests tutorial"
+subtitle: "Step by step - Adding a new PrestaShop UI test"
+date:  2020-09-30 12:00:00
+authors: [boubkerbribri]
+icon: icon-github-alt
+image: /assets/images/theme/meta-logo-build.png
+tags: [community, contribution, tests, automation]
+---
+
+One year ago, the QA team started to develop a new test framework, based on [Puppeteer](https://github.com/puppeteer/puppeteer), [Mocha](https://mochajs.org/), and [Chai](https://www.chaijs.com/).
```

Comment:
there is an article from Simon about that if I remember well. that's worth it to add a link to it.

Reply:
link added :+1: